operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2679,6 +2679,9 @@ metaMonitoring:
     # -- ServiceMonitor relabel configs to apply to samples before scraping
     # https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#relabelconfig
```

```suggestion
    # -- ServiceMonitor relabel configs to apply to targets before scraping
    # https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.RelabelConfig
```

I just realized these are slightly out of date/inaccurate. Hope you don't mind including them in this PR

---

Not at all, modified 👍